gettsim/tests/test_alg2.py

```diff
@@ -47,14 +47,20 @@
 
 
 years = [2006, 2009, 2011, 2013, 2016, 2019]
-tax_policy_data = load_tax_benefit_data()
+
+
+@pytest.fixture
+def input_data():
+    file_name = "test_dfs_alg2.csv"
+    out = pd.read_csv(f"{ROOT_DIR}/tests/test_data/{file_name}")
```

Comment:
Same issue as above. pathlib lets you concat paths with ``/``.

Reply:
See above